src/blueprints/tv2/action-factories/ActionFactory.ts

```diff
@@ -0,0 +1,5 @@
+export abstract class ActionFactory {
+  protected sanitizeStringForId(value: string): string {
+    return value.replaceAll(/\s/g, '_').replaceAll('/', '_')
```

Comment:
Isn't the `g` in the regular expression unnecessary, since `replaceAll` is used?

Reply:
I think you are right. I just copied the regex from one of the ActionFactories